css/src/components/popover.scss

```diff
@@ -0,0 +1,35 @@
+.popover {
+	display: inline-block;
+	position: relative;
+
+	.popover-summary {
+		color: $hyperlink;
```

Comment:
Remove `color`? If we're going for composability here, we don't need it.

Reply:
Well, I did that to make it clear that's element is clickable. Asked Courtny about it, by the end of the week design should be done so I'm planning to update it then.

Reply:
Yes, the issue is composability though. If we don't set anything here, we're free to style it however we like with other classes.

Reply:
PR looks good, just waiting on this one.

Reply:
Sounds good, I'll update the default one to link so there is an example of that view!

Reply:
Updated!